src/Text/Pandoc/Readers/Mdoc/Macros.hs

```diff
@@ -0,0 +1,185 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE OverloadedStrings #-}
+{- |
+   Module      : Text.Pandoc.Readers.Mdoc.Macros
+   Copyright   : 
+   License     : GNU GPL, version 2 or above
+
+   Maintainer  : Evan Silberman <evan@jklol.net>
+   Stability   : WIP
+   Portability : portable
+
+-}
+module Text.Pandoc.Readers.Mdoc.Macros (isParsedMacro, isCallableMacro) where
+
+import Data.Set (member, fromList, Set)
+import Data.Text
+import Language.Haskell.TH.Syntax (lift)
+
+isParsedMacro :: Text -> Bool
+isParsedMacro a = member a parsedMacros
+
+isCallableMacro :: Text -> Bool
+isCallableMacro a = member a callableMacros
+
+parsedMacros :: Set Text
+parsedMacros = $(lift (fromList [
```

Comment:
Am I doing a pointless optimization here by making these sets and the map in Standards.hs at compile time? I suspect it's pointless, and I don't have any evidence that it isn't. But for now it's still here.

Reply:
Pretty sure it's pointless.  I think the compiler is smart enough to create the map before runtime.
You could always benchmark the alternative to see if it makes a difference.  If it doesn't, it would be nice to avoid the TH.

Reply:
I'm just gonna drop it.